content/source/docs/cloud/api/run.html.md

```diff
@@ -83,6 +83,9 @@ Key path                    | Type   | Default | Description
 ----------------------------|--------|---------|------------
 `data.attributes.is-destroy` | bool | false | Specifies if this plan is a destroy plan, which will destroy all provisioned resources.
 `data.attributes.message` | string | "Queued manually via the Terraform Enterprise API" | Specifies the message to be associated with this run.
+`data.attributes.refresh` | bool | true | Specifies whether or not to refresh the state before a plan.
+`data.attributes.refresh-only` | bool | false | Specifies whether the run should ignore configuration changes and only refresh the state.
```

Comment:
Hmm, still kind of sounds like Terraform might possibly perform some actions on me. How about,

```suggestion
`data.attributes.refresh-only` | bool | false | Whether this run should use the refresh-only plan mode, which will refresh the state without modifying any resources. Mutually exclusive with `is-destroy`.
```

Reply:
Much nicer!